### ISSUE TITLE
Getting prop error using package for vue3 and vuetify

### ISSUE THREAD

**Issue by BrownAJKE** [kyvg/vue3-notification#36]
**Describe the bug**
I have added the component on App.vue
`<VApp>
      <VMain>
        <RouterView />
        <Notifications
          group="success"
          position="bottom right"
        />
      </VMain>
    </VApp>`

**To Reproduce**

**Screenshots**
<img width="1429" alt="Screenshot 2022-11-28 at 12 07 55" src="https://user-images.githubusercontent.com/65741746/204237860-b0be486e-7ca3-4baa-91f7-84f59d05d028.png">

What I'm I doing wrong?

**Comment by kyvg** [comment]
Warns looks like you forgot to provide `"notifications"` prop for your `App.vue` (or `VApp.vue` or `VMain.vue`). This lib has no prop `"notifications"`

Are you sure you added following code to your `main.ts`?
```
import { createApp } from 'vue'
import Notifications from '@kyvg/vue3-notification'

const app = createApp({...})
app.use(Notifications)
```

**Comment by BrownAJKE** [comment]
I have added that code on my main.js

**Comment by BrownAJKE** [comment]
<img width="939" alt="Screenshot 2022-11-28 at 12 44 43" src="https://user-images.githubusercontent.com/65741746/204245808-47c08a78-1c9c-4d3a-b99b-6aabf83ff21a.png">
Here's a screenshot of the same

**Comment by kyvg** [comment]
I can't reproduce that error, but I found that `Notifications` is not equal to `notifications`. Can you add `name` option like that?

```app.use(Notifications, { name: "Notifications" })```

**Comment by kyvg** [comment]
> I can't reproduce that error, but I found that `Notifications` is not equal to `notifications`. Can you add `name` option like that?
> 
> `app.use(Notifications, { name: "Notifications" })`

I'm sorry, it's should be `componentName`

 `app.use(Notifications, { componentName: "Notifications" })`

**Comment by kyvg** [comment]
Fixed case sensitive component name in `v2.7.0`

**Comment by BrownAJKE** [comment]
I'm still getting the same error, I dont know why
